### PR TITLE
Create Game Session for Multiplayer

### LINKED ISSUE #21: As a user I want a word selection feature so that an external user can select the word that two competing users need to try to find
Related to epic task #5 

User-story points:
8

### LINKED ISSUE #111: As a user playing as an admin in multiplayer i want to be able to set the word so that others can attempt to guess it
related to #5

User-story Points:
5

### PR DESCRIPTION
**Pull request by robinjonker**
Closes #111 Closes #21
[pull_request_140.md](https://github.com/witseie-elen4010/2022-group-lab-011/files/8839827/pull_request_140.md)

### CODE REVIEW

File: public/index.js [hunk 1]
```diff
@@ -0,0 +1,64 @@
+document.addEventListener('DOMContentLoaded',() => {
+
+    window.onresize = onWindowResize
```
Comment (gouws-j) [inline public/index.js:3]: What does added index do? Is it cosmetic?

Comment (robinjonker) [reply public/index.js:3]: It is the script that was within the HTML page, separated it to two files

**Review by gouws-j — APPROVED** [review]
Good stuff! One step closer to getting multiplayer to work.

**Review by gouws-j — APPROVED** [review]

**Comment by gouws-j** [comment]
[pull_request_checklist_111_21.md](https://github.com/witseie-elen4010/2022-group-lab-011/files/8839852/pull_request_checklist_111_21.md)

**Pull request merged**